tools/autograph-monitor/monitor.go

```diff
@@ -94,8 +94,7 @@ func main() {
 	} else {
```

Comment:
Oh, this isn't the code that's being run, right? Should we add a wrapping around `Handler` that logs the errors it returns to help us along?

Reply:
Oh, I see your comment in Slack saying that the error logging is fixed by the upgrade!